### DISCUSSION TITLE
Localized country names

### DISCUSSION THREAD

**Discussion by Lerbiq** [InvoiceShelf/InvoiceShelf#167] [Ideas]
**Is your feature request related to a problem? Please describe.**
The problem is that completely localized invoices are currently not possible because of some fields not being localized. For example the country - Czech Republic in my case.

**Describe the solution you'd like**
The ability to use localized country names. Either directly localized based on the language set in system preferences or localized to the native language of the selected country (like Czech Republic -> Česká republika).

**Describe alternatives you've considered**
Editing the country name in the database directly.

**Comment by rihards-simanovics** [comment]
This is a sound suggestion, but my concern is that it will blow up the size of already huge locale file to be even bigger. I think we first need to re-consider how we deal with locales going forward, as like it's been already shown in v2.0.0 that it's extremely easy to change something in locales and break everything.

As it is likely me, who will be doing the frontend for the app going forward, if time allows, I'll re-write it in React and will be strongly considering breaking down of the locale file into a folder instead of files. This should make things a little more manageable linguistically speaking, as you would be able to separate various parts of locales into their own manageable/digestible segments.
